# /**/takeoff_behaviour: 
/**:
  ros__parameters:
    default_takeoff_altitude: 1.0  # [m]
    default_takeoff_speed: 0.4     # [m/s]
    takeoff_height_threshold: 0.1  # [m]
